sensorprocess/sensorprocess_test.go

```diff
@@ -283,39 +283,39 @@ func TestAddSensorReading(t *testing.T) {
 	cf := cartofacade.Mock{}
 
 	config := Config{
-		Logger:      logger,
-		CartoFacade: &cf,
-		DataRateMs:  200,
-		Timeout:     10 * time.Second,
+		Logger:            logger,
+		CartoFacade:       &cf,
+		LidarDataRateMsec: 200,
+		Timeout:           10 * time.Second,
 	}
 	ctx := context.Background()
 
 	t.Run("returns error when lidar GetData returns error, doesn't try to add sensor data", func(t *testing.T) {
-		sensors := []string{"invalid_sensor"}
+		cam := map[string]string{"name": "invalid_lidar"}
```

Comment:
why are we creating a map, when only the value of one of its keys are used... Could we please remove all the `cam` maps in this test file & change them to just be strings?